odxtools/odxlink.py

```diff
@@ -28,6 +30,11 @@ class OdxDocFragment:
     doc_type: DocType
 
 
+class OdxDocContext(NamedTuple):
```

Comment:
I prefer to have this class in a separate `odxdoccontext.py` file. In my mind it is not directly connected to the ODXLINK reference mechanism...

Reply:
done